blogposts/livebloggig-gophercon-2019.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Sourcegraph liveblogging at GopherCon 2019!
+author: Ryan Blunden - Developer Advocate at Sourcegraph
+publishDate: 2019-07-22T00:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-liveblogging-at-gophercon-2019
+heroImage: https://res.cloudinary.com/bizzaboprod/image/upload/c_crop,g_custom,f_auto/v1546982777/uj2i17bmkcyfuvxakuem.png
+published: true
+---
+
+<p class="text-center"><img alt="GopherCon 2019 Banner" src="https://res.cloudinary.com/bizzaboprod/image/upload/w_740,c_fit,fl_any_format,q_auto:best/mjkpm9uwwnknzs4u7b6s.png" class="h5"></p>
+
+Sourcegraph is the standard developer platform for code search and navigation at many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.
+
+For the third year, Sourcegraph is proud to host the official liveblog for GopherCon. We're looking for attendees who want to contribute - is that you?
+
+## Why liveblog
+
+Liveblogging is an excellent way to get more involved with the Go community, support the presenters, and get your name on an artifact of GopherCon. Sign up for multiple sessions to keep your options open and help make sure the schedule gets filled.
+
+## How it works
+
+Liveblogging is a live stream in written form. It's capturing the essential parts of a talk, converting that into Markdown including images is cool too), then making a pull request against our [website repository](https://github.com/sourcegraph/about). Once your pull request is merged, it will live on the [GopherCon blog](/go) in less than 5 minutes.
```

Comment:
Do we have  a liveblog template they can clone? Should also make sure they know what directory/where to put the file, and if we have any guidelines.

```suggestion
Liveblogging is a live stream in written form. It's capturing the essential parts of a talk, and converting that into Markdown (including images is awesome!). Next, make a pull request against our [website repository](https://github.com/sourcegraph/about) to get it hosted on [sourcegraph.com](https://about.sourcegraph.com/blog/). Once your pull request is merged, it will be live on the [GopherCon blog](/go) in less than 5 minutes.
```